Handler/Sites.hs

```diff
@@ -85,17 +85,25 @@ embedExample msiteId = do
 
 siteForm :: Maybe Site -> Form Site
 siteForm msite = renderBootstrap3 BootstrapBasicForm $ Site
-    <$> areq textField (bfl "Name") (siteName <$> msite)
-    <*> areq textField (bfl "Base URL") (siteBaseUrl <$> msite)
-    <*> areq textField (bfl "RSS Author") (siteRssAuthor <$> msite)
-    <*> areq textField (bfl "RSS Title") (siteRssTitle <$> msite)
-    <*> areq htmlField (bfl "RSS Description") (siteRssDescription <$> msite)
-    <*> areq textField (bfl "RSS Language") (siteRssLanguage <$> msite)
+    <$> areq textField (fs "Name" "my-site") (siteName <$> msite)
+    <*> areq textField (fs "Base URL" "https://example.com") (siteBaseUrl <$> msite)
+    <*> areq textField (fs "Comments Feed Author" "My Site") (siteRssAuthor <$> msite)
+    <*> areq textField (fs "Comments Feed Title" "My Site Comments") (siteRssTitle <$> msite)
```

Comment:
That's fine, it just means I can't do the form in this simple way. I'll have to accept a datatype with fewer fields, then build a full Site out of the result.

Which fields do you think they should enter: just Name, URL, Language?

I'm going to have to rename our sites I think ("Latest comments on robots-production" doesn't sound right).